src/Forum/client.ts

```diff
@@ -0,0 +1,49 @@
+import fetch, { Response } from 'node-fetch'
+import { env } from 'decentraland-commons'
+import { ForumPost } from './Forum.types'
+
+export type CreateSuccess = {
+  id: number
+  name: string
+  username: string
+  topic_slug: string
+  display_username: string
+  created_at: string
+  cooked: string
+  errors: undefined
+}
+
+export type CreateError = {
+  action: string
+  errors: string[]
+}
+
+type CreateResponse = CreateSuccess | CreateError
+
+export const FORUM_URL = env.get('REACT_APP_FORUM_URL', '')
```

Comment:
Is this variable being used outside this file?

Reply:
Not really! I'll delete the export